dali/ft/filecopy.cpp

```diff
@@ -3259,18 +3262,23 @@ void FileSprayer::updateTargetProperties()
                 {
                     FilePartInfo & curSource = sources.item(idx);
                     RemoteFilename &remoteFile = curSource.filename;
-                    splitAndStoreFileInfo(newRecord, remoteFile, idx, false);
+                    splitAndCollectFileInfo(newRecord, remoteFile, fileNames, idx, false);
+
                 }
+                newRecord->setProp("@name", "[MULTI]");
```

Comment:
It isn't necessarily a multi file spray because in this branch is it? i.e. just because !distributedSource doesn't mean multi file...

All sprays from a landing zone will be !distributedSource - so as it stands a standard 1 file spray would be labelled "[MULTI]".

I think you need to say : if (sources.ordinality()>1) ... set "[MULTI]" otherwise set @name to the fileNames string.

Reply:
Right